addons/binding/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/handler/ExecHandler.java

```diff
@@ -53,13 +55,14 @@
     public static final String TIME_OUT = "timeout";
     public static final String COMMAND = "command";
     public static final String TRANSFORM = "transform";
-    public static final String AUTORUN = "autorun";
+    public static final String RUN_ON_INPUT = "runOnInput";
+    public static final String REPEAT_ENABLED = "repeatEnabled";
 
     // RegEx to extract a parse a function String <code>'(.*?)\((.*)\)'</code>
     private static final Pattern EXTRACT_FUNCTION_PATTERN = Pattern.compile("(.*?)\\((.*)\\)");
 
     private ScheduledFuture<?> executionJob;
-    private String lastInput;
+    private String lastInput = "";
```

Comment:
I do not like compiler warnings, nor putting Ignore annotations ;-) But I agree with yr comment